mantle/harness/harness_test.go

```diff
@@ -76,6 +76,8 @@ func TestContextCancel(t *testing.T) {
 }
 
 func TestSubTests(t *testing.T) {
+	term := os.Getenv("TERM")
+	os.Unsetenv("TERM")
```

Comment:
Let's add a comment here explaining why we unset TERM.

Also do we really need to restore TERM? The tests will run in a process and then finish (i.e. it doesn't affect the calling process).